### PR TITLE
ARIA 1.2 updates to Allowed ARIA roles, states and properties table

### LINKED ISSUE #192: Update global ARIA attributes
In ARIA 1.2 the following ARIA attributes will no longer be global:

* aria-disabled
* aria-errormessage
* aria-haspopup
* aria-invalid

ARIA in HTML needs to be updated to reflect this.

### LINKED ISSUE #251: checkbox and switch should allow aria-expanded
### Current

https://w3c.github.io/html-aria/#allowed-aria-roles-states-and-properties

* The checkbox role does not allow aria-expanded.
* The switch role does not allow aria-expanded.

However, the ARIA Editor's Draft allows them.

* https://w3c.github.io/aria/#checkbox
* https://w3c.github.io/aria/#switch

### Proposed

Add aria-expanded in the Supported Properties column for checkbox and switch.

### PR DESCRIPTION
**Pull request by scottaohara**
This PR replaces the closed #198 PR

Updates the informative cells of the Allowed ARIA roles, states and properties to reflect ARIA 1.2 allowances

closes #192 
closes #251

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/html-aria/pull/300.html" title="Last updated on May 13, 2021, 10:00 PM UTC (fadba54)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/html-aria/300/813397c...fadba54.html" title="Last updated on May 13, 2021, 10:00 PM UTC (fadba54)">Diff</a>

### CODE REVIEW

**Review by lastthyme — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -3519,42 +3534,93 @@ <h2>
...
+            <th tabindex="-1" id="index-aria-checkbox">
+              <a data-cite="wai-aria-1.2#checkbox">`checkbox`</a>
+            </th>
+            <td>
+              <a data-cite="wai-aria-1.2#aria-checked">`aria-checked`</a>
+            </td>
+            <td>
+              <ul>
+                <li>
+                  <a data-cite="wai-aria-1.1#aria-disabled">`aria-disabled`</a></li>
```
Comment (patrickhlauke) [inline index.html]: all of these should probably also point to `wai-aria-1.2` rather than `wai-aria-1.1`

**Comment by patrickhlauke** [comment]
seems we need to fundamentally agree whether we want to wholesale send things to 1.2, or if we're happy with a weird mix of 1.1 and 1.2 (or if we want to leave it all for now until this goes to REC, and only tackle it afterwards)

**Comment by scottaohara** [comment]
This is a pr for afterwards

**Comment by patrickhlauke** [comment]
you ok with me doing a big find/replace to normalise everything to 1.2?

**Comment by scottaohara** [comment]
That’s fine. But not going to merge it till later. There’s a bunch of other 1.2 work that’d need to be done as well that would cause normative updates that we don’t want to do now.

**Comment by scottaohara** [comment]
merging #324 made this PR largely unnecessary.  updating the links to ARIA 1.2 can be done as a new PR when that spec is in Rec.

**Pull request closed without merging**